### PR TITLE
add test helper file (mainly for test usage)

### PR DESCRIPTION
**Pull request by Arieltk1**

### CODE REVIEW

File: services/install_opereto_worker_libs/opereto/helpers/test_helper.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+import uuid
+import shutil
+
+def zipfolder(zipname, target_dir):
```
Comment (drorrusso) [inline services/install_opereto_worker_libs/opereto/helpers/test_helper.py]: I would consider adding this one to the worker/services/install_opereto_worker_libs/opereto/utils/osutil.py (we already have zip functions there). In the test helper I think we should only keep test-related functions.

Comment (Arieltk1) [reply services/install_opereto_worker_libs/opereto/helpers/test_helper.py]: Will do. Sending fix soon

File: services/install_opereto_worker_libs/opereto/utils/osutil.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from zipfile import ZipFile as zip
 import hashlib
 import platform
-from opereto.utils.shell import run_shell_cmd
```
Comment (drorrusso) [inline services/install_opereto_worker_libs/opereto/utils/osutil.py]: Why? 
Relative imports less recommended. Less readable and clear and can break easily. What if you have another module called shell in the python standard lib for instance?

Comment (Arieltk1) [reply services/install_opereto_worker_libs/opereto/utils/osutil.py]: Fixed

**Pull request merged**